torch/optim/optimizer.py

```diff
@@ -164,16 +164,33 @@ def update_group(group, new_group):
             update_group(g, ng) for g, ng in zip(groups, saved_groups)]
         self.__setstate__({'state': state, 'param_groups': param_groups})
 
-    def zero_grad(self):
-        r"""Clears the gradients of all optimized :class:`torch.Tensor` s."""
+    def zero_grad(self, set_to_none=False):
```

Comment:
Can you update `optimizer.pyi` as well to reflect this change? Just add the new arg and type there.